traffic_ops/experimental/ui/build/README.md

```diff
@@ -11,10 +11,13 @@
 * Build the RPM
 
     ```
-    $ cd traffic_control/build
-    $ ./docker-build.sh -r https://github.com/apache/incubator-trafficcontrol.git -b master traffic_ops_v2
+    $ cd traffic_ops/experimental/ui/build
+    $ docker build -t tov2-build .
+    $ docker run -v $(pwd)/artifacts:/artifacts -e GITREPO=https://github.com/apache/incubator-trafficcontrol.git -e BRANCH=master tov2-build
```

Comment:
this is going to get merged into master to can we get rid of the tov2-build part?